Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -15,14 +16,14 @@ namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
     /// </summary>
     public static class HealthVaultVitalStatisticsCodes
     {
-        public const string System = "http://healthvault.com/vocabularies/vital-statistics";
+        public static readonly string System = VocabularyUris.HealthVaultVocabulariesUri;
 
-        internal static Coding BodyWeight = new Coding()
+        internal static readonly Coding s_bodyWeight = new Coding()
```

Comment:
The rules we published say: 

We use  _camelCase  for internal and private fields and use  readonly  where possible. Prefix instance fields with  _ , static fields with  s_  and thread static fields with  t_ . When used on static fields,  readonly  should come after  static  (i.e.  static readonly  not  readonly static ).

This is an internal static field. 


---
In reply to: [125125550](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r125125550) [](ancestors = 125125550)